bucket/mp3tag.json

```diff
@@ -52,6 +55,7 @@
         "   if (!(is_admin)) {error \"$app requires admin rights to $cmd\"; break}",
         "   Start-Process 'regsvr32' -Wait -Verb 'RunAs' -ArgumentList @('/u', \"$dir\\Mp3tagShell32.dll\", '/s')",
         "   Start-Process 'regsvr32' -Wait -Verb 'RunAs' -ArgumentList @('/u', \"$dir\\Mp3tagShell.dll\", '/s')",
+        "   Stop-Process -Name 'explorer' -Force; Start-Sleep -Seconds 2",
```

Comment:
Unacceptable

Reply:
How is this _Unacceptable_ to you?

Reply:
If I don't use **Stop-Process** to restart explorer, then unregistering the context wouldn't happen.

Reply:
what if I have some explorer windows open that I am working on? that's why I always say to let the user execute certain commands/scripts manually

Reply:
Should I edit the script to break the **uninstallation** process if they have the context menu registered? Then tell them how to uninstall the context menu?

Reply:
I think all of your changes can be revert except for the persistence part. if you think the context menu is important, you should create a copy of it in the nonportable bucket.

the app works just as fine even without a context menu, and this is the expected behaviour for most portable apps. I don't see why you would want to change that.

i'm not saying that apps installed via Scoop cannot get the full experience. but these are extras setup steps in fact. I don't want it in my context menu, even when I use the app.(AND this is the reason most people use Scoop instead of winget/choco/official installer of app)

Reply:
Fixed it.